README.md

```diff
@@ -33,39 +33,44 @@ the Pull Request Conversation:
 
 **Required**. Must be in form of `github_token: ${{ secrets.github_token }}`.
 
-### `level`
+### `working_directory`
+
+Optional. Directory to run the action on, from the repo root.
+The default is `.` ( root of the repository).
+
+### `reviewdog_log_level`
 
 Optional. Report level for reviewdog [`info`,`warning`,`error`].
 It's same as `-level` flag of reviewdog.
 The default is `error`.
 
-### `reporter`
+### `reviewdog_reporter`
```

Comment:
I prefer not to change the flags for reviewdog. Adding tflint_ (<linter_name>_) prefix to tflint flags should be enough.

It's also important to be consistent with other reviewdog actions in the reviewdog org.

Reply:
Fair enough! I thought using `reviewdog_*` and `tfsec_*` would add clarity, but I don't feel strongly enough about it. I agree that consistency with other actions is a better reason.

Done in https://github.com/reviewdog/action-tfsec/pull/15/commits/42655a3d6fe909a123b72dfd6b030dc0dc732f37